docs/architecture/event-flows-proposal.md

```diff
@@ -0,0 +1,77 @@
+# Eventing flow proposal
```

Comment:
It shouldn't be called proposal

Reply:
good, point, we shoudl call as a Draft/proposal only section with External Bus